test/integration/etcd/server.go

```diff
@@ -70,12 +77,25 @@ func StartRealMasterOrDie(t *testing.T, configFuncs ...func(*options.ServerRunOp
 		t.Fatal(err)
 	}
 
+	saSigningKeyFile, err := ioutil.TempFile("/tmp", "key")
```

Comment:
nit: name the file so it is clear it is a test key (`insecure_test_key` or something), since we're setting permission file permissions on it. applies to all the places we're persisting the test keys

Reply:
done